<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1213</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Shot</string>
    </property>
   </widget>
   <widget class="QLabel" name="imglabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>781</width>
      <height>561</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>image display</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="houghResolution">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.000010000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QComboBox" name="threshold">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>cv.ADAPTIVE_THRESH_GAUSSIAN_C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cv.ADAPTIVE_THRESH_MEAN_C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cv.THRESH_BINARY + cv.THRESH_OTSU</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cv.THRESH_BINARY</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hough Resolution</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Hough Min Dist.</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="MINRAD">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2147483647</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Hough param1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Hough param2</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="param1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2147483647</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="param2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2147483647</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Hough Min Radius</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Hough Max Radius</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="houghMinrad">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2147483647</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="houghMaxrad">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2147483647</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Circle Range</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="circleRange">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>350</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2147483647</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>px</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gaussian Std. Devi.</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="blurStdDev">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="maximum">
     <double>2147483647.999989986419678</double>
    </property>
    <property name="singleStep">
     <double>0.000010000000000</double>
    </property>
    <property name="value">
     <double>5.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Dis. StdDevi.</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="distStdDev">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="maximum">
     <double>2147483647.999989986419678</double>
    </property>
    <property name="singleStep">
     <double>0.000001000000000</double>
    </property>
    <property name="value">
     <double>0.300000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Crop Gsn. StdDevi.</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="blurStdDev_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>410</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="maximum">
     <double>2147483647.999989986419678</double>
    </property>
    <property name="singleStep">
     <double>0.000010000000000</double>
    </property>
    <property name="value">
     <double>3.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance Between</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="distance">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>440</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="maximum">
     <double>2147483647.999989986419678</double>
    </property>
    <property name="singleStep">
     <double>0.000010000000000</double>
    </property>
    <property name="value">
     <double>300.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="CamAngle">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>470</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2147483647</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cam. Angle</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FilePath">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>186</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="FindFile">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>26</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>540</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="ValueOutput">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>10</y>
      <width>181</width>
      <height>561</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Value Output</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>blurStdDev</tabstop>
  <tabstop>houghResolution</tabstop>
  <tabstop>MINRAD</tabstop>
  <tabstop>param1</tabstop>
  <tabstop>param2</tabstop>
  <tabstop>houghMinrad</tabstop>
  <tabstop>houghMaxrad</tabstop>
  <tabstop>threshold</tabstop>
  <tabstop>circleRange</tabstop>
  <tabstop>distStdDev</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
